modules/menu.js

```diff
@@ -0,0 +1,39 @@
+export const pDate = document.querySelector('#date');
+export const list = document.querySelector('.nav-sections:nth-child(1)');
+export const addNew = document.querySelector('.nav-sections:nth-child(2)');
+export const contact = document.querySelector('.nav-sections:nth-child(3)');
+export const booksSection = document.querySelector('#all-awesome-books');
+export const formSection = document.querySelector('#add-new-book');
+export const contactSection = document.querySelector('#contact');
+
+// Rest of the code...
+
+export function showForm() {
+  booksSection.classList.add('hide');
+  formSection.classList.remove('hide');
+  contactSection.classList.add('hide');
+
+  list.classList.remove('change-color');
+  contact.classList.remove('change-color');
+  addNew.classList.add('change-color');
+}
+
+export function showContact() {
+  booksSection.classList.add('hide');
+  formSection.classList.add('hide');
+  contactSection.classList.remove('hide');
+
+  list.classList.remove('change-color');
+  addNew.classList.remove('change-color');
+  contact.classList.add('change-color');
+}
+
+export function showBooks() {
+  booksSection.classList.remove('hide');
+  formSection.classList.add('hide');
+  contactSection.classList.add('hide');
+
+  addNew.classList.remove('change-color');
+  contact.classList.remove('change-color');
+  list.classList.add('change-color');
+}
```

Comment:
- [ ] Please I highly suggest refactoring the functions in your code base to ES6 arrow functions per project requirements. 😄 

Example 

```
export const showForm = () => {
}

export const showContact = () => {
}

export const showBooks = () => {
}
```